pkg/postage/batchservice/batchservice_test.go

```diff
@@ -54,12 +54,14 @@ func (m *mockBatchListener) HandleCreate(b *postage.Batch, topUpAmount *big.Int)
 	return nil
 }
 
-func (m *mockBatchListener) HandleTopUp(_ []byte, _ *big.Int) {
+func (m *mockBatchListener) HandleTopUp(id []byte, newBalance *big.Int) error {
```

Comment:
If the method params are not used inside its body they should be renamed to `_`.